.circleci/config.yml

```diff
@@ -119,7 +122,7 @@ workflows:
   nightly_build:
     triggers:
       - schedule:
-          cron: "00 16 * * *"
+          cron: "30 16 * * *"
```

Comment:
why is this changed?

Reply:
no reason, this is caused by versioning.